pkg/ssh/ssh.go

```diff
@@ -242,20 +243,58 @@ func RunSSHUpgradeSubnetEVM(host *models.Host, subnetEVMBinaryPath string) error
 	)
 }
 
-func RunSSHCopyMonitoringDashboards(host *models.Host, monitoringDashboardPath string, customGrafanaDasboardPath string) error {
-	// TODO: download dashboards from github instead
+func replaceCustomVarDashboardValues(monitoringDashboardPath, customGrafanaDashboardFileName, chainID string) error {
+	sedScript := fmt.Sprintf("s/\"text\": \"CHAIN_ID_VAL\"/\"text\": \"%[1]v\"/g; s/\"value\": \"CHAIN_ID_VAL\"/\"value\": \"%[1]v\"/g; s/\"query\": \"CHAIN_ID_VAL\"/\"query\": \"%[1]v\"/g", chainID)
+	cmd := exec.Command("sed", "-i", "-e", sedScript, customGrafanaDashboardFileName)
```

Comment:
Does this work on Mac? I thought you had to do `-i ""`

Or is this just run on linux?